vocabularies/party-relationship.ttl

```diff
@@ -0,0 +1,85 @@
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix pyrl: <http://linked.data.gov.au/def/party-relationship/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/party-relationship> a owl:Ontology, skos:ConceptScheme .
+    dct:created "2020-03-24"^^xsd:date ;
+    dct:creator <http://linked.data.gov.au/org/gsq> 
+    dct:modified "2020-03-25"^^xsd:date ;
+    dct:source "Compiled by the Geological Survey of Queensland." ;
+    skos:definition "The relationships that exist between companies, businesses, and individuals."@en ;
+    skos:hasTopConcept pyrl:adviser,
+        pyrl:contractor,
+        pyrl:director,
+        pyrl:employee,
+        pyrl:joint-venture,
+        pyrl:owner,
+        pyrl:parent,
+        pyrl:subsidiary,
+        pyrl:volunteer ;
+    skos:prefLabel "Party Relationship"@en .
+
+<http://linked.data.gov.au/org/gsq>
+    a sdo:Organization ;
+    sdo:name "Geological Survey of Queensland" ;

```

Comment:
End with full stop

Reply:
Copied as per Nick Cars bulk vocabulary update

<http://linked.data.gov.au/org/gsq>
    a sdo:Organization ;
    sdo:name "Geological Survey of Queensland" ;
.

Reply:
While that syntax works, Nick should clean his code. The semi-colon is used to copy the subject to the next line.

Reply:
removed last semi-colon, moved fullstop up